packages/fab/index.js

```diff
@@ -47,14 +52,15 @@ export class Fab extends React.Component {
           className={this.classes}
           ref={initRipple}
           {...otherProps}>
-          {this.addIconClassToAllChildren()}
+          {this.renderIcon()}
         </button>
     );
   }
 }
 
 Fab.propTypes = {
   mini: PropTypes.bool,
+  icon: PropTypes.element,
   className: PropTypes.string,
   children: PropTypes.oneOfType([
```

Comment:
remove children from proptypes. I think you're planning on using `label` or `text` props instead.